Modules/core/Controller/CoreaccountController.php

```diff
@@ -69,16 +69,10 @@ public function indexAction() {
                 $modelPeningAccounts->add($id_user, $form->getParameter("id_space"));
 
                 $mailer = new MailerSend();
-                $mail_from = getenv('MAIL_FROM');
-                if (!empty($mail_from)) {
-                    $from = $mail_from;
-                }
-                else {
-                    $from = "support@platform-manager.com";
-                }
+                $from = Configuration::get('smtp_from');
                 $fromName = "Platform-Manager";
                 $toAdress = $form->getParameter("email");
-                $subject = CoreTranslator::Account($lang);
+                $subject = CoreTranslator::AccountCreatedSubject($lang);
```

Comment:
wen user is created in a space, user is autojoined

Either:
* account creation email is sent AND a "join" mail is send prefixed by space name in subject
* account creation email only is sent, and spaceName should be prefixed in email (who created his account)

Reply:
So I modified CoreaccountController.php  but it appeared, when I tried to test it, that I can't find the way to access it by the UI. When trying to type directly the url "{host}/corecreateaccount", i'm getting a 403.
I don't know in which case it is used, but it's now meant to work with use of the new Email.php class

Reply:
For self user registration?
It is disabled in ui, for routes i don t know as it does not appear in ui

Reply:
Yes. The original modification I made was only because I had changed Account() into AccountCreatedSubject()  - for a semantic purpose - in CoreTranslator and checked if any files were impacted